src/Materials/Textures/prePassRenderTarget.ts

```diff
@@ -0,0 +1,120 @@
+import { MultiRenderTarget, IMultiRenderTargetOptions } from "./multiRenderTarget";
+import { Engine } from "../../Engines/engine";
+import { RenderTargetTexture } from './renderTargetTexture';
+import { Scene } from "../../scene";
+import { PostProcess } from "../../PostProcesses/postProcess";
+import { ImageProcessingPostProcess } from "../../PostProcesses/imageProcessingPostProcess";
+import { Nullable } from "../../types";
+
+/**
+ * A multi render target designed to render the prepass.
+ * Prepass is a scene component used to render information in multiple textures
+ * alongside with the scene materials rendering.
+ * Note : This is an internal class, and you should NOT need to instanciate this.
+ * Only the `PrePassRenderer` should instanciate this class.
+ * It is more likely that you need a regular `MultiRenderTarget`
+ */
+export class PrePassRenderTarget extends MultiRenderTarget {

```

Comment:
Shouldn't we add a `@hidden` as this class is not meant to be used by the user?

Reply:
yep